src/Components/Title/styles.js

```diff
@@ -0,0 +1,10 @@
+import {css} from 'styled-components';
+import colors from "../../themes/colors";
+
+export const TitleStyles = css`
+    font-family: "Times New Roman", Times, serif;
```

Comment:
Let's add font-face and base theme with ThemeProvider. See like implemented this in project I've given you